kits/quant-x-terminal/apps/src/index.css

```
@@ -0,0 +1,13 @@
+body {
```

Stylelint (17.14.0)

[error] Unknown rule scss/at-rule-no-unknown. Did you mean at-rule-no-unknown?

(scss/at-rule-no-unknown)